packages/expo-build-properties/src/android.ts

```diff
@@ -1,3 +1,4 @@
+import { ManifestQuery } from '@expo/config-plugins/build/android/Manifest';
```

Comment:
should export `ManifestQuery` type from @expo/config-plugins so that we could just import the type from `expo/config-plugins`
https://github.com/expo/expo/blob/3ddd8761abfd9d1cfe4c16966c908d68fe5b077f/packages/%40expo/config-plugins/src/Plugin.types.ts#L197